conf/accounts.routes

```diff
@@ -5,7 +5,7 @@ GET /resume-draft-return/:id
 GET /view-sent-return/:submissionId                                             uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.accounts.homepage.HomePageController.viewSentReturn(submissionId: String)
 GET /pay-total-outstanding                                                      uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.accounts.homepage.HomePageController.payTotalAmountLeftToPay()
 GET /exit/report-another-way                                                    uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.accounts.homepage.HomePageController.exitForSubsequentReturn()
```

Comment:
can `/exit/report-another-way` be deleted? I'm not sure...

Reply:
Looks like it's still used https://github.com/hmrc/cgt-property-disposals-frontend/blob/57833eb11d99d2dce71037ac7e4a33831261c808/app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/representee/RepresenteeController.scala#L811